finite_difference/src/argument_mod.f90

```diff
@@ -103,6 +103,11 @@ module argument_mod
 !> Kernel requires the vertical grid spacing of the grid.
 !! Requires/assumes that this quantity is constant.
 integer, public, parameter :: GO_GRID_DY_CONST = 15
+integer, public, parameter :: GO_GRID_X_MIN_INDEX=16
```

Comment:
Please add comments specifying what these quantities are - i.e. do they refer to the internal region, the whole grid or something else?

Reply:
Done, but there is a bit of an issue: the constants being declared in dl_esm_inf, while their usage is specified in PSyclone. This should (imho) go into a config file ... in the distant future I guess ;)